docs/topics/i18n/timezones.txt

```diff
@@ -481,7 +481,7 @@ Setup
 
    Yes. When time zone support is enabled, Django uses a more accurate model
    of local time. This shields you from subtle and unreproducible bugs around
-   Daylight Saving Time (DST) transitions.
+   daylight saving time (DST) transitions.
```

Comment:
As far as I could see, on the twitter thread linked in description, they use `Daylight saving time`.
```suggestion
   Daylight saving time (DST) transitions.
```

Reply:
Yes, but only because it is first word of a sentence. 

Extract from the book. 

![Screenshot_20210730-201835.jpg](https://user-images.githubusercontent.com/39445562/127702056-231c0d93-1826-4c43-a22b-ab37e8e5c183.jpg)

Reply:
thanks for the clarification, it is fine then, sorry for the noise.

Reply:
No problem. It was a good challenge. 👍